pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -625,6 +625,22 @@ func (c ClusterConfig) IsFargateEnabled() bool {
 	return len(c.FargateProfiles) > 0
 }
 
+func (c ClusterConfig) HasNodes() bool {
+	count := 0
+	if c.ManagedNodeGroups != nil {
+		for _, m := range c.ManagedNodeGroups {
+			count += m.GetDesiredCapacity()
```

Comment:
Hang on, why are you counting these? How come the `len(bla) > 0` isn't enough? Do you need the count anywhere?

Reply:
oh right yeah duh :facepalm: ill refactor. silly jake